### PR TITLE
add support for scanning archived flags

### PR DESCRIPTION
**Pull request by atrakh**
Minor release: archived flags should have code references

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,10 +3,12 @@ module github.com/launchdarkly/ld-find-code-refs
 go 1.13
 
 require (
+	github.com/antihax/optional v1.0.0
 	github.com/go-git/go-git/v5 v5.1.0
+	github.com/golang/protobuf v1.4.3 // indirect
 	github.com/hashicorp/go-retryablehttp v0.5.0
 	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334
-	github.com/launchdarkly/api-client-go v0.0.0-20181031211650-fe4f1d017b6e
+	github.com/launchdarkly/api-client-go v3.9.0+incompatible
```
Comment (atrakh) [inline go.mod:11]: Updated api-client-go to the latest version to bring in support for the `archived` filter.

File: README.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ We do not yet have repositories set up for our linux packages, but we do upload
 
 ##### Ubuntu
 
-This shell script can be used to download and install `ag` and `ld-find-code-refs` on Ubuntu.
+This shell script can be used to download and install `ld-find-code-refs` on Ubuntu.
```
Comment (atrakh) [inline README.md:70]: This part of the readme was outdated.

File: internal/ld/ld.go [hunk 3]
```diff
@@ -92,14 +93,22 @@ func InitApiClient(options ApiOptions) ApiClient {
 
 func (c ApiClient) GetFlagKeyList() ([]string, error) {
 	ctx := context.WithValue(context.Background(), ldapi.ContextAPIKey, ldapi.APIKey{Key: c.Options.ApiKey})
-	flags, _, err := c.ldClient.FeatureFlagsApi.GetFeatureFlags(ctx, c.Options.ProjKey, nil)
+
+	flags, _, err := c.ldClient.FeatureFlagsApi.GetFeatureFlags(ctx, c.Options.ProjKey, &ldapi.GetFeatureFlagsOpts{Summary: optional.NewBool(true)})
```
Comment (hjacobs1) [inline internal/ld/ld.go:97]: What's the purpose of `Summary: optional.NewBool(true)`?

Comment (atrakh) [reply internal/ld/ld.go:97]: Ah, should have noted this change. This filter option will make the api requests for feature flags a big faster by omitting flag configuration data that `ld-find-code-refs` doesn't need

**Review by hjacobs1 — APPROVED** [review]
LGTM! Just had one question about how this works.

File: internal/ld/ld.go (hunk 3 above)
Comment (hjacobs1) [reply internal/ld/ld.go:97]: That makes sense. Thanks!

**Pull request merged**